app/assets/src/helpers/url.js

```diff
@@ -27,3 +29,9 @@ export const getURLParamString = params => {
     .map(pair => pair.join("="))
     .join("&");
 };
+
+export const copyShortUrlToClipboard = async url => {
+  url = url || window.location.href;
+  const shortUrl = await shortenUrl(url);
+  copy(window.location.origin + "/" + shortUrl.unique_key);
```

Comment:
Form the snapshot noticed this unique is a little short (4 chars). Can we increase the size a bit?

Reply:
The default is 5 chars, which gives us 36^5 = 60M keys. I can configure it now, or we can wait until it is a problem and add more chars. What do you think?

Reply:
If the change is backwards-compatible, its fine...
But if we are not expiring I would add a 2/3 more characters.